portfolio/src/main/webapp/quiz.js

```diff
@@ -2,40 +2,91 @@
  * Question data type.
  */
 class Question {
-  constructor(question, answers, correct) {
+  constructor(question, answers, correctAnswerIndex) {
     this.question_ = question;
     this.answers_ = answers;
     this.correctAnswerIndex_ = correctAnswerIndex;
   }
 
   getQuestion() {
-    return question_;
+    return this.question_;
   }
 
   getAnswers() {
-    return answers_;
+    return this.answers_;
   }
 
   getCorrectAnswerIndex() {
-    return correctAnswerIndex_;
+    return this.correctAnswerIndex_;
   }
 }
 
+// create array of quiz questions 
+const questions = [];
+questions.push(
+    new Question('What\'s my GitHub username?',
+		['iolawale', 'ihsan', 'olawale314', 'ihsan314'], 3));
+questions.push(
```

Comment:
Consider naming this parameter. Example:

From style guide: “Parameter name” comments should be used whenever the value and method name do not sufficiently convey the meaning.

Example: 
/* correctAnswerIndex= */ 3

Reply:
Ok, I will do that.